source/EsUuid.cpp

```diff
@@ -31,6 +31,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <uuid/uuid.h>
 #endif
 
+
```

Comment:
```suggestion


```
even more!

Reply:
My IDE is doing some funky stuff, apparently.